src/Server.cpp

```diff
@@ -8,8 +8,11 @@
 #include <fstream>
 #include <sstream>
 
-Server::Server(ServerCfg& cfg, ServerConfig& gen_cfg) :_cfg(cfg), _gen_cfg(gen_cfg) { }
-Server::Server(const Server& copy) :_cfg(copy._cfg), _gen_cfg(copy._gen_cfg) { }
+Server::Server( ServerCfg& cfg, ServerConfig& gen_cfg ) :_cfg( cfg ), _gen_cfg( gen_cfg ) {
+}
+
+Server::Server( const Server& copy ) :_cfg( copy._cfg ), _gen_cfg( copy._gen_cfg ) {
+}
```

Comment:
Don't really see the use these style updates if it's not consistent. (Look at the destructor right under this)

Reply:
These kinds of changes are not necessarily things I "correct", instead were made when trying to get the program to run / understand what does what.